.github/workflows/_sandbox.yaml

```diff
@@ -1,41 +1,99 @@
-name: "~Sandbox"
+name: CI
```

Comment:
Please don't forget to reset the sandbox before making the PR ready for merge.
```
git checkout main _sandbox.yaml
```
should do

Reply:
Yeah, I know.
Done